### PR TITLE
reverting to original feedstock rendering

### PR DESCRIPTION
**Pull request by buijennifer**
- Build No. 5
- Restoring feedstock files to match templates from conda smithy repo
- Showing that when "upload_on_branch" is not defined in conda-forge.yml, files follow original templates
- Example for PR

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
     git_rev: v{{ version }}
 
 build:
-    number: 4 
+    number: 5 
     script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
     script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

**Review by mrakitin — CHANGES_REQUESTED** [review]
This is a similar update as in https://github.com/buijennifer/event-model-feedstock/pull/11#pullrequestreview-262090473.

File: recipe/meta.yaml (hunk 1 above)
Comment (buijennifer) [reply recipe/meta.yaml]: I made the changes you suggested.

**Comment by buijennifer** [comment]
Newly built packages were not uploaded with changes:

https://dev.azure.com/buijennifer/nsls2/_build/results?buildId=210&view=logs&jobId=4f922444-fdfe-5dcf-b824-02f86439ef14&taskId=937c195f-508d-5135-dc9f-d4b5730df0f7&lineStart=858&lineEnd=859&colStart=1&colEnd=1

**Pull request merged**